modules/ROOT/examples/live-demos/typography/index.html

```diff
@@ -0,0 +1,28 @@
+<textarea id="advanced-typography">
+
+<h3>Using the Advanced Typography plugin</h3>
+
+<ol>
+<li>Select the text of the short narrative below.</li>
+<li>Press the typography toolbar button to run the <em>Advanced Typography</em> plugin’s en-US typography rules.</li>
+<li>See the text change from typewriter-centric typesetting to typographer-friendly typesetting.</li>
+</ol>
+
+<h3>Short narrative</h3>
+
+<p>"It's over there", she said.</p>
+
+<p>"No. There" -- -> -- she pointed.</p>
```

Comment:
FYI: Double hyphens are not actually changed to a single em-dash. 
What actually happens:
1. Second hyphen is removed as a duplicate;
2. First hyphen is transformed to a em-dash.

Reply:
FWIW, I’d argue this is a hole in the rules. The double-hyphen (whether set open or closed) as em-dash substitute is a long-standing emergent norm of typewriter-centric typesetting.

Rarer is the triple-hyphen (`---`) as em-dash substitute. I’ve no formal studies or surveys on this front but, in my experience, people who use the triple-hyphen tend to also use the double-hyphen as a substitute to the en-dash.

By contrast, folk like me who use the double-hyphen as a desperation device when proper dashes are not available use the hyphen set open as the en-dash substitute.

This does, of course, constitute a complication for anyone seeking to automate the _typewriter typography – professional typography_ workflow. 

All this said, given the rules act to, if only by fortuitous accident, replace a double-hyphen with an em-dash, I’m inclined to  leave the example be.

Double-hyphens are replete in material produced by people with no typesetting knowledge, and having them be transformed into em-dashes normally improves both the presentation and the grammatical accuracy of said material.